### PR TITLE
GA of Compare changes feature

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Advanced CI / Compare changes feature is moving from Preview to GA status.

- Updated CI jobs page 
- Updated Advanced CI page 
- Updated Compare changes tab in the Run visibility page  
- Updated Accounts settings page related to Adv CI

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x] Need PM review

### CODE REVIEW

File: website/docs/docs/deploy/advanced-ci.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ sidebar_label: "Advanced CI"
 description: "Advanced CI enables developers to compare changes by demonstrating the changes the code produces."
 ---
 
-# Advanced CI <Lifecycle status="preview" />
+# Advanced CI <Lifecycle status="enterprise"/>
```
Comment (nghi-ly) [inline website/docs/docs/deploy/advanced-ci.md:8]: @reubenmc : should this have an enterprise pill or just compare changes section below?

File: website/docs/docs/cloud/account-settings.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ To use, select the **Enable partial parsing between deployment runs** option fro
 
 <Lightbox src="/img/docs/deploy/example-account-settings.png" width="85%" title="Example of the Enable partial parsing between deployment runs option" />
 
-## Account access to Advanced CI features <Lifecycle status="beta" />
+## Account access to Advanced CI features <Lifecycle status="enterprise" />
```
Comment (nghi-ly) [inline website/docs/docs/cloud/account-settings.md:42]: @reubenmc : just flagging since i'm not sure if i applied the pill correctly here.

Comment (reubenmc) [reply website/docs/docs/cloud/account-settings.md:42]: Looks good to me!

File: website/docs/docs/deploy/advanced-ci.md (hunk 1 above)
Comment (reubenmc) [reply website/docs/docs/deploy/advanced-ci.md:8]: Let's just call all of Adv CI enterprise. I think we need to do some naming work here between compare changes vs advanced CI. But this is good for now!

File: website/docs/docs/deploy/advanced-ci.md [hunk 3]
```diff
@@ -5,12 +5,12 @@ sidebar_label: "Advanced CI"
...
 
 [Continuous integration workflows](/docs/deploy/continuous-integration) help increase the governance and improve the quality of the data. Additionally for these CI jobs, you can use Advanced CI features, such as [compare changes](#compare-changes), that provide details about the changes between what's currently in your production environment and the pull request's latest commit, giving you observability into how data changes are affected by your code changes. By analyzing the data changes that code changes produce, you can ensure you're always shipping trustworthy data products as you're developing.
 
-:::tip Preview feature
-The compare changes feature is currently available as a [preview](/docs/dbt-versions/product-lifecycles#dbt-cloud) in dbt Cloud. dbt Labs plans to provide additional Advanced CI features in the near future. More info coming soon.
+:::tip More features
```
Comment (reubenmc) [inline website/docs/docs/deploy/advanced-ci.md:12]: <3

**Review by reubenmc — COMMENTED** [review]
LGTM

**Review by reubenmc — APPROVED** [review]
Whoops, meant to approve the first time!

**Review by nataliefiann — APPROVED** [review]
Hiya @nghi-ly 

Thanks for creating this. It was a really good read. I have no recommendations or suggestions - I've approved this for you.

Kind Regards
Natalie

**Pull request merged**